Analysis/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
 #@authors K. Smith, S. V. Paulauskas, C. R. Thornsberry
-option(PAASS_BUILD_UTKSCAN "Build utkscan" OFF)
```

Comment:
Might we still want this option? Some users may not want utkscan and it takes some time to compile.

Reply:
That option is still there. Look a little further in the file on line 34.

Reply:
Sorry, I missed it.